R/reporter-progress.R

```diff
@@ -78,13 +82,19 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
     },
 
     show_status = function(complete = FALSE) {
+      time <- proc.time()
```

Comment:
Does this get used somewhere?

Reply:
No it is vestigial, thanks for catching it.